src/test/resources/features/complaints/ComplaintsDraft.feature

```diff
@@ -99,6 +99,18 @@ Feature: Complaints Draft
     And an Escalation note should be visible in the timeline showing the submitted reason for the cases escalation
     And the read-only Case Details accordion should contain all case information entered during the "Service Draft" stage
 
+  @ComplaintsWorkflow @ComplaintsRegression
+  Scenario: User is able to escalate a BF case to WFM at Service Draft stage
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "Draft" stage
+    And I load and claim the current case
+    And I add a "DRAFT" type document to the case
+    And I escalate the case to WFM at Service Draft stage
```

Comment:
Maybe go back and update this step def so it's something like 'And I escalate the complaint case to WFM at the Draft stage'